openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityLineage/LineageNodeLabelV1.tsx

```diff
@@ -223,16 +225,20 @@ const EntityFooter = ({
         {childrenInfoDropdownLabel}
       </Button>
       <TestSuiteSummaryContainer node={node} />
-      <IconButton
-        className={classNames(
-          'only-show-columns-with-lineage-filter-button',
-          isOnlyShowColumnsWithLineageFilterActive && 'active'
-        )}
-        data-testid="lineage-filter-button"
-        disabled={isEditMode}
-        onClick={handleOnlyShowColumnsWithLineage}>
-        <FilterIcon height={20} width={20} />
-      </IconButton>
+      <Tooltip
+        placement="right"
```

Comment:
Can we move placement to bottom? in the screen recording the tooltip overlaps with another node.